authui/src/authflowv2/base.css

```diff
@@ -101,6 +101,9 @@
 
     /* background */
     --color-bg-base: #1c1c1e;
+
+    /* text */
+    --color-text-base: #ffffff;
```

Comment:
1. Remove color-bg-base
2. Add color-surface-primary
3. Add color-white
4. Add color-black
5. Change existing places to use color-surface-primary or color-white, on your own judgement.

Reply:
I think existing places which uses `--color-bg-base` should point to `color-white` or `color-neutral-700`. `color-surface-primary` and `color-white` is not on the same level, and I prefer not to introduce `color-surface-primary`

Reply:
idk if this is relevant, the text button for light and dark mode can be set separately in the future.
<img width="457" alt="image" src="https://github.com/user-attachments/assets/24cddb3a-bc42-4264-8ac0-fd300d6d6e5f">